openapi.yaml

```diff
@@ -11394,6 +11451,25 @@ components:
         - expiresAt
         - status
         - links
+    OptOutOption:
+      title: OptOutOption
+      type: object
+      description: |-
+        The opt-out option payload.
+      x-examples:
+        rmsUnsubscribeUrl: '@@web_link@@'
+        oneClickApiMapping: "unsubscribe"
+      x-tags:
+        - Email
+      properties:
+        rmsUnsubscribeUrl:
+          type: string
+          example: "@@web_link@@#unsubscribe"
+          description: This is the unsubscribe URL of your message.
+        oneClickApiMapping:
+          type: string
+          example: "unsubscribe"
+          description: This is the API mapping code or ID.
```

Comment:
maybe add more details on what the API mapping code is for

Reply:
I'll resolve this comment :) @lnachor-whispir